react/components/Slider.tsx

```diff
@@ -86,6 +86,7 @@ const Slider: FC<Props> = ({
         className={`w-100 ${handles.sliderTrackContainer} ${
           shouldUsePagination ? 'overflow-hidden' : 'overflow-x-scroll'
         }`}
+        data-testid="slider-track-container"
```

Comment:
Do we really need this? Isn't any other way to get this element in tests?

Reply:
Not anymore ;)